aurielle-auriellep/hw_6/app.js

```diff
@@ -0,0 +1,51 @@
+$(function() {
+    _500px.init({
+        sdk_key: YOUR_API_KEY,
+    });
+
+    $('#login').on('click', function() {
+        _500px.login();
+
+    });
+    _500px.on('authorization_obtained', function() {
+        $('.sign-in-view').hide();
+        $('.image-results-view').show();
+        if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition(function(position) {
+
+                let lat = position.coords.latitude;
+                let long = position.coords.longitude;
+                console.log('lat: ' + lat);
+                console.log('long: ' + long);
+
+                let radius = '25mi';
+                let searchOptions = {
+                    geo: `${lat},${long},${radius}`,
+                    only: 'Landscapes',
+                    rpp: 28,
+                    sort: 'highest_rating',
+                };
+                _500px.api('/photos/search', searchOptions, function(response) {
+                    const photoArray = response.data.photos;
+                    for (const photo of photoArray) {
+                        $('.images').append(`<img src='${photo.image_url}' alt='${photo.description}' class='image'>`)
```

Comment:
Although the content in photos can still be altered, you made a good choice in assigning that data to an immutable variable. Constants will take you a long way, and I can see you're already working that best practice into your repertoire.

The only thing I'd suggest here is including more comments in your code for organization and explanation of what each piece is doing. Not only is it helpful for other developers to figure out what's going on at a glance, but future-Aurielle will benefit from it when coming back to refactor.